ragen/env/sokoban/config.py

```
@@ -13,9 +13,12 @@ class SokobanEnvConfig:
     dim_x: Optional[int] = None
     dim_y: Optional[int] = None
     render_mode: str = "text"
+    observation_format: str = "grid"
 
     def __post_init__(self):
```

Ruff (0.14.4)

Missing return type annotation for special method `__post_init__`

(ANN204)